lib/doorkeeper/orm/active_record/application.rb

```diff
@@ -12,6 +12,14 @@ class Application < ActiveRecord::Base
     validates :uid, uniqueness: true
     validates :redirect_uri, redirect_uri: true
 
+    validates_each :scopes, if: :enforce_scopes? do |record, attr, value|
+      valstr = value.to_s
```

Comment:
No need to do `to_s` because ActiveModel / ActiveRecord already done type-casting for us.